react/src/components/masthead/components/SprkMastheadSelector/SprkMastheadSelector.js

```diff
@@ -218,14 +247,19 @@ SprkMastheadSelector.propTypes = {
    */
   additionalTriggerTextClasses: PropTypes.string,
   /**
-   * Assigned to the `data-analytics` attribute serving as a unique selector for outside libraries to capture data.
+   * Assigned to the `data-analytics` attribute serving as
+   * a unique selector for outside libraries to capture data.
    */
   analyticsString: PropTypes.string,
-  /** Configuration object that builds the dropdown */
+  /** Configuration object that builds the dropdown. */
   choices: PropTypes.shape({
-    /** A node to render at the foot of the dropdown menu */
+    /**
+     * A function supplied that will run when a choice is made.
+     */
+    choiceFunction: PropTypes.func,
```

Comment:
This was not included in the Prop Validation for some reason. Adding it in here because ESLint caught it.